great-idea/index.html

```diff
@@ -16,7 +16,78 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="container">
+
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </nav>
+        <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+
```

Comment:
This logo could also be wrapped in an anchor tag the same way you do it on the services page.